pkg/apis/networking/validation/validation.go

```diff
@@ -362,32 +375,83 @@ func validateHTTPIngressPath(path *networking.HTTPIngressPath, fldPath *field.Pa
 	default:
 		allErrs = append(allErrs, field.NotSupported(fldPath.Child("pathType"), *path.PathType, supportedPathTypes.List()))
 	}
-
-	allErrs = append(allErrs, validateIngressBackend(&path.Backend, fldPath.Child("backend"), opts)...)
+	allErrs = append(allErrs, validateIngressBackend(&path.Backend, fldPath.Child("backend"), opts, requestGV)...)
 	return allErrs
 }
 
+// portFieldName returns the name of the field used for an
+// ingress service port in the provided GroupVersion.
```

Comment:
// numberPortField returns the field path to a service port number field
// relative to a backend struct in the provided GroupVersion